packages/service/lib/schema.js

```diff
@@ -314,6 +314,12 @@ const server = {
               }
             }
           ]
+        },
+        requestCert: {
+          type: 'boolean'
+        },
+        rejectUnauthorized: {
+          type: 'boolean'
         }
       },
       additionalProperties: false,
```

Comment:
Wondering if we sohuld have `additional Propertis` set to `false` here, considering that users might want to setup also others props for https://nodejs.org/api/tls.html#new-tlstlssocketsocket-options

Reply:
cc @mcollina WDYT?